contrib/scripts/env-setup.py

```diff
@@ -77,18 +78,38 @@ def get_version(version):
     return VersionConfig(f'nightly+{version}', f'pytorch-dev{version}',
                          get_py_version(), cuda_version)
 
-  version_regex = re.compile(r'^(\d+\.)+\d+$')
-  if not version_regex.match(version):
+
+  if not VERSION_REGEX.match(version):
     raise ValueError(f'{version} is an invalid torch_xla version pattern')
   return VersionConfig(
     version, f'pytorch-{version}', get_py_version(), cuda_version)
 
 
 def install_vm(version, apt_packages, is_root=False):
   dist_bucket = DIST_BUCKET
+
   if version.cuda_version:
+    # Distributions for GPU runtime
+    # Note: GPU wheels available from 1.11
     dist_bucket = os.path.join(
       DIST_BUCKET, 'cuda/{}'.format(version.cuda_version.replace('.', '')))
+  else:
+    # Distributions for TPU runtime
+    # Note: this redirection is required for 1.11 & nightly releases
+    # because the current 2 VM wheels are not compatible with colab environment.
+    if version.wheels == 'nightly':
+      dist_bucket = os.path.join(DIST_BUCKET, 'colab/')
+    elif 'nightly+' in version.wheels:
+      build_date = datetime.strptime( version.wheels.split('+')[1], '%Y%m%d')
+      if build_date >= NEW_VERSION:
+        dist_bucket = os.path.join(DIST_BUCKET, 'colab/')
```

Comment:
are we missing an else here? nightly <= 2021/09/01(rough date) should work using default wheel. I doubt anyone will want that through.

Reply:
Hey @JackCaoG , thanks for asking. This should work as intended -- as the default path is already pointing to the default wheels.